### DISCUSSION TITLE
Function for quick editing snippets

### DISCUSSION THREAD

**Discussion by omegahh** [L3MON4D3/LuaSnip#713] [Q&A]
I switched to LuaSnip from UtilSnips recently. And also to json-formatted snippets like VScode. I am wondering that it there any function for quick editing snippets for current buffer. Just like `let g:UltiSnipsEditSplit="vertical"` in UltiSnips

**Comment by L3MON4D3** [comment]
Something like this https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#edit_snippets?

**Reply by omegahh** [reply]
Excellent! I only remember I have seen it somewhere. Thank you!
